library/src/morpheus-master/sass/modules/tool/samigo/_samigo.scss

```
@@ -421,4 +421,9 @@
 		display: none;
 	}
 
+	.exitPreviewButton {
+		margin-top: 5px;
+		margin-left: 80px;
+	}
```

The spacing here seems a bit arbitrary. The button should really be right-aligned, but more work on the banners needs to be done first (which I'm working on for another PR). 

For now, I think a standard space between the end of the text and the button will be sufficient. 

```suggestion
	.exitPreviewButton {
		margin-left: $standard-space;
	}
```

---

Thank you, @fostersdesign. I'v made this change that you suggested and made a new commit.
Greetings.

---

Thanks.